metrics/fds_linux.go

```diff
@@ -48,7 +48,7 @@ func currentFDs() int {
 	}
 }
 
-func direntNamlen(dirent *syscall.Dirent) int {
+func direntNamelen(dirent *syscall.Dirent) int {
```

Comment:
Seemed like a reasonable change, but perhaps not?

Reply:
Let's keep it. It's named after the field.